Composer/packages/client/src/pages/design/ManifestEditor.tsx

```diff
@@ -74,6 +76,10 @@ const styles = {
 const helpLink =
   'https://docs.microsoft.com/en-us/azure/bot-service/skills-write-manifest-2-1?view=azure-bot-service-4.0';
 
+const isExternalLink = (url: string): boolean => {
```

Comment:
suggestion: put the function to utilities and reuse here and in `Composer/packages/client/src/recoilModel/dispatchers/botProjectFile.ts`